sklearn/cluster/_agglomerative.py

```diff
@@ -747,6 +747,13 @@ class AgglomerativeClustering(ClusterMixin, BaseEstimator):
 
         .. versionadded:: 0.21
 
+    compute_distances : bool, default=False
+        Computes distances even if no distance_threshold is used. This can be
```

Comment:
The "even if no distance_threshold is used" does not seem add too more information and may be a little confusing:

```suggestion
        Computes distances between clusters. This can be
```

Reply:
Added "between clusters" in 61e5c5a459d1f4d76912417ee965d11039f82f45.

I have note removed "even if no distance_threshold is used" because IMHO it is important to clarify what is the typical use case for this parameter, i.e. precisely providing the `distances_` attribute even if the strategy is based on `n_clusters` rather than `distance_threshold`.

If you disagree with this I will of course follow your suggestions.

Reply:
Maybe just rephrase to:

=> even if distance_threshold is not used

Reply:
Fixed in 1e53c30382ac55272f6818e9c78e8d85f2ebcf4b.